examples/nffPktgen/testing/minPktLoss.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2018-2019 Intel Corporation.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+        "flag"
+        "fmt"
+        "os"
+        "os/signal"
+        "sync/atomic"
+        "time"
+
+        "github.com/intel-go/nff-go/examples/nffPktgen/generator"
+        "github.com/intel-go/nff-go/flow"
+        "github.com/intel-go/nff-go/packet"
+)
+
+type IpPort struct {
+        Index       uint16
+        packetCount uint64
+        bytesCount  uint64
+}
+
+type HandlerContext struct {
+        port *IpPort
+}
+
+func (hc HandlerContext) Copy() interface{} {
+        return HandlerContext{
+                port: hc.port,
+        }
+}
+
+func (hc HandlerContext) Delete() {
+}
+
+func main() {
+        var (
+                speed            uint64
+                tgtLoss          float64
+                trafDelay        uint
+                genConfig, cores string
+                inPort           uint
+                outPort          uint
+        )
+        flag.Uint64Var(&speed, "speed", 120000000, "speed of fast generator, Pkts/s")
+        flag.Float64Var(&tgtLoss, "target loss", 0.005, "target packet loss percentage, use 0.001 for 1 percent")
```

Comment:
Can you please make it in human readable percents and check that the value is between 0 and 100?